src/Features/LanguageServer/Protocol/Handler/Definitions/GoToDefinitionHandler.cs

```diff
@@ -10,14 +10,14 @@ namespace Microsoft.CodeAnalysis.LanguageServer.Handler
 {
     [Shared]
     [ExportLspMethod(LSP.Methods.TextDocumentDefinitionName)]
-    internal class GoToDefinitionHandler : GoToDefinitionHandlerBase, IRequestHandler<LSP.TextDocumentPositionParams, LSP.Location[]>
+    internal class GoToDefinitionHandler : GoToDefinitionHandlerBase, IRequestHandler<LSP.TextDocumentPositionParams, object>
     {
         [ImportingConstructor]
         public GoToDefinitionHandler(IMetadataAsSourceFileService metadataAsSourceFileService) : base(metadataAsSourceFileService)
         {
         }
 
-        public async Task<LSP.Location[]> HandleRequestAsync(Solution solution, LSP.TextDocumentPositionParams request,
+        public async Task<object> HandleRequestAsync(Solution solution, LSP.TextDocumentPositionParams request,
```

Comment:
work item to track whether we want to leave things as object or start use UnionType people are talking about on LSP

Reply:
Sure.  But for these they will have to be object to be backwards compatible with older liveshare clients.

Reply:
https://github.com/dotnet/roslyn/projects/45#card-21810125